pea/formatter.py

```diff
@@ -23,7 +20,7 @@ def __init__(self, *args):
 		type(self).instance = self
 
 	def setOutputStream(self, stream):
-		type(self).stream = stream
+		type(self).stream = colorama.initialise.wrap_stream(stream, True, False, False, True)
```

Comment:
I guess it would be helpful if you could use keyword arguments here for all those obvious booleans ;)